src/services/amqpClient.js

```diff
@@ -0,0 +1,50 @@
+require('dotenv').config()
+const connection = require('amqp-connection-manager').connect(process.env.QUEUE_URL)
+
+connection.on('connect', () => {
+  console.log('Connected to amqp Broker')
+})
+
+connection.on('disconnect', () => {
+  console.error('Disconnected from amqp Broker')
+})
+
+function connectWatcherToQueue({ queueName, cb }) {
+  connection.createChannel({
+    setup(channel) {
+      const sendToQueue = data =>
+        channel.sendToQueue(queueName, Buffer.from(JSON.stringify(data)), { persistent: true })
```

Comment:
I see that `node-amqp-connection-manager` has a `json` option that seems to automatically stringify and parse JSONs in the messages. Would it make sense to use it here?

Reply:
I think it make sense to use the `json` option. I'll give it a try and test it

Reply:
Tested and the `json` option automatically stringify data on `sendToQueue` but you still have to parse the data manually on the consumer. Anyway, it's an improvement so I'll add it.